cylc/flow/scheduler_cli.py

```diff
@@ -234,6 +234,12 @@ def get_option_parser(is_restart, add_std_opts=False):
         metavar="PLUGIN_NAME", action="append", dest="main_loop"
     )
 
+    parser.add_option(
+        "--abort-if-any-task-fails", "--fragile",
```

Comment:
Not so sure about `--fragile` as an alias.

Reply:
I'm not sure about `fragile` as an alias either (or @kinow 's variant).  It has connotations that aren't necessarily accurate. An abort is a controlled clean shut down, not falling apart like a broken vase :thinking:  

`--abort-if-any-task-fails` is verbose, but it won't be a heavily used option, and it is nicely unambiguous.